internal/tools/gittar/metrics/collector.go

```diff
@@ -165,20 +166,39 @@ func (c *Collector) IterateRepos(req apistructs.GittarListRepoRequest) ([]*Perso
 		gitRepository.Url = conf.GittarUrl() + "/" + repo.Path
 		gitRepository.IsExternal = repo.IsExternal
 
-		branch, err := gitRepository.GetDefaultBranch()
+		branches, err := gitRepository.GetBranches()
 		if err != nil {
-			logrus.Warningf("failed to get default branch for repo: %s, err: %v, use branch: %s as default", repo.Path, err, defaultBranch)
-			branch = defaultBranch
+			logrus.Warningf("failed to get all branch for repo: %s, err: %v, use branch: %s as default", repo.Path, err, defaultBranch)
+			branches = append(branches, defaultBranch)
 		}
-		sourceCommit, err := gitRepository.GetBranchCommit(branch)
-		if err != nil {
-			logrus.Errorf("failed to get commit for branch: %s", branch)
-			continue
+		var allSourceCommit []*gitmodule.Commit
+		for _, branch := range branches {
+			if !strutil.HasPrefixes(branch, "master", "Master", "main", "Main", "dev", "Dev", "develop", "Develop", "release", "Release") {
```

Comment:
define `master`, `main` and son on in a variable
like:
var (
  targetBranch = []string{"master","main","dev"}
)

Reply:
ok,yeah,thanks